lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2409,39 +2409,64 @@ def bar(self, left, height, zs=0, zdir='z', *args, **kwargs):
         return patches
 
     def bar3d(self, x, y, z, dx, dy, dz, color=None,
-              zsort='average', *args, **kwargs):
-        '''
-        Generate a 3D bar, or multiple bars.
-
-        When generating multiple bars, x, y, z have to be arrays.
-        dx, dy, dz can be arrays or scalars.
-
-        *color* can be:
-
-         - A single color value, to color all bars the same color.
-
-         - An array of colors of length N bars, to color each bar
-           independently.
-
-         - An array of colors of length 6, to color the faces of the
-           bars similarly.
+              zsort='average', shade=True, *args, **kwargs):
+        """Generate a 3D barplot.
+
+        This method creates three dimensional barplot where the width,
+        depth, height, and color of the bars can all be uniquely set.
+
+        Parameters
+        ----------
+        x, y, z : array-like
+            The coordinates of the anchor point of the bars.
+
+        dx, dy, dz : scalar or array-like
+            The width, depth, and height of the bars, respectively.
+
+        color : sequence of valid color specifications, optional
+            The color of the bars can be specified globally or
+            individually. This parameter can be:
+
+              - A single color value, to color all bars the same color.
+              - An array of colors of length N bars, to color each bar
+                independently.
+              - An array of colors of length 6, to color the faces of the
+                bars similarly.
+              - An array of colors of length 6 * N bars, to color each face
+                independently.
+
+            When coloring the faces of the boxes specifically, this is
+            the order of the coloring:
+
+              1. -Z (bottom of box)
+              2. +Z (top of box)
+              3. -Y
+              4. +Y
+              5. -X
+              6. +X
+
+        zsort : str, optional
+            The z-axis sorting scheme passed onto
+            :func:`~mpl_toolkits.mplot3d.art3d.Poly3DCollection`
+        shape : bool, optional (default = True)
+            When true, the dark sides of the bars (relative to the
+            plot's source of light), shaded.
```

Comment:
typo: shape --> shade

Also, the description is missing a verb. Additionally, I think there is supposed to be a blank line between the zsort entry and the shade entry?